namd/tests/library/001_orientationangle_harmonic-fixed/namd-version.txt

```diff
@@ -1,3 +1,3 @@
-Info: NAMD 3.1alpha3 for Linux-x86_64-multicore
-colvars: Initializing the collective variables module, version 2025-08-22.
-colvars: Using NAMD interface, version "2025-05-12".
+Info: NAMD 3.0.2 for Linux-x86_64-multicore
+colvars: Initializing the collective variables module, version 2024-06-04 (patch 1).
```

Comment:
Note: I updated this manually to reflect that this is the second patch update to NAMD 3.0

Reply:
I didn't know this patch number before...